### ISSUE TITLE
Spawners

### ISSUE THREAD

**Issue by Joannou1** [GLServers/GLD#8]
Needs input.

If we can get the mob stacker plugin, that will solve the lag department...
...But the effects on PVP and the economy still remain to be seen.

If we have plugins that utilize materials from mobs from grinders, then spawners will have to become harder to get.

**Comment by Kladdkaka** [comment]
If we find a way to modify the enchantment algorithm, then I think it would be a great idea that silk touch is harder to get. And you can mine spawners by using a silk touch pickaxe on a spawner, and the pickaxe breaks instantly & it cost a amount of money, and also we could make so it's a 33% chance it works :P I think that would be quite balanced

**Comment by Joannou1** [comment]
Yeah that would be really cool to do that... need to note it down to look into it!
to the misc post

**Comment by Joannou1** [comment]
Currently finding a way to modify the enchantment algorithm may be difficult...
But if it can be done, we could allow for mineable spawners to be a thing, but set this really low:

 # Percentage of dropping the spawner when mined
silkDropChance: 100

**Comment by Kladdkaka** [comment]
I found a way to change the algorithm :p
